core/src/transfer/data/kzg_blob.rs

```diff
@@ -8,11 +8,78 @@
 //! type.
 
 use alloc::string::ToString;
-use c_kzg::{ethereum_kzg_settings, Blob as KzgBlob};
+use c_kzg::{ethereum_kzg_settings, Blob as KzgBlob, Bytes48};
 
 use super::{BlobData, BlobSidecar, Error};
 
 impl BlobData {
+    /// Creates a default `c_kzg::KzgSettings` for KZG operations.
+    ///
+    /// This function initializes and returns a static reference to a
+    /// `c_kzg::KzgSettings` instance, which can be used for Ethereum KZG
+    /// operations. If no `precompute` value is provided, the default value
+    /// of `8` is used, which is recommended for Ethereum's KZG settings.
```

Comment:
where is this recommended?
In the link provided below it says:
```
the precompute parameter only affects the performance of compute_cells_and_kzg_proofs and recover_cells_and_kzg_proofs. If your application does not use these functions, we recommend using precompute=0. 
```

It seems to me that we don't use those functions...